styles.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
- [ ] Please kindly fix the background image. It's not present in the view. This is probably because there is a problem with the URL 
![image](https://user-images.githubusercontent.com/37901469/164703325-b89fa56f-8d69-4e95-bf35-96f4c41c9466.png)